src/plugins/home/public/application/components/home.js

```diff
@@ -150,28 +146,29 @@ export class Home extends Component {
     const stackManagement = this.findDirectoryById('stack-management');
 
     return (
-      <EuiPage restrictWidth={1200} data-test-subj="homeApp">
-        <EuiPageBody className="eui-displayBlock">
-          <EuiScreenReaderOnly>
-            <h1>
-              <FormattedMessage id="home.welcomeHomePageHeader" defaultMessage="Kibana home" />
-            </h1>
-          </EuiScreenReaderOnly>
-          <EuiSpacer />
-          <EuiPageHeader>
-            <EuiPageHeaderSection>
-              <EuiTitle size="m">
+      <Fragment>
+        <header className="homPageHeader">
+          <EuiFlexGroup gutterSize="none">
```

Comment:
Regarding the `gutterSize="none"` prop, we should probably include some gutter space for smaller viewport sizes.

Reply:
There's a whole host of work needed here for responsiveness. I shied away from that to keep this PR readable 😬